doc/articles/uno-development/building-uno-ui.md

```diff
@@ -15,6 +15,19 @@ This article explains how to build Uno.UI locally, for instance if you wish to c
 - Install (**Tools** / **Android** / **Android SDK manager**) all Android SDKs starting from 7.1 (or the Android versions `TargetFrameworks` [list used here](https://github.com/unoplatform/uno/blob/master/src/Uno.UI.BindingHelper.Android/Uno.UI.BindingHelper.Android.csproj))
 - Run [Uno.Check](https://github.com/unoplatform/uno.check) on your dev machine to setup .NET 6 Android/iOS workloads
 
+## Recommended Windows hardware
```

Comment:
This is consistent with the rest of the document and also adheres to markdownlint:

https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md022---headers-should-be-surrounded-by-blank-lines

```suggestion
## Recommended Windows hardware

```